.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+
+    steps:
+      - uses: actions/stale@v10
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "This issue has been inactive for 60 days. Please comment if it is still relevant or it will be closed in 14 days."
+          stale-pr-message: "This pull request has been inactive for 60 days. Please update it or it will be closed in 14 days."
+          stale-issue-label: "no-issue-activity"
+          stale-pr-label: "no-pr-activity"
+          days-before-stale: 60
+          days-before-close: 14
+          exempt-labels: "pinned,important"
```

actionlint (1.7.10)

[error] input "exempt-labels" is not defined in action "actions/stale@v10". available inputs are "any-of-issue-labels", "any-of-labels", "any-of-pr-labels", "ascending", "close-issue-label", "close-issue-message", "close-issue-reason", "close-pr-label", "close-pr-message", "days-before-close", "days-before-issue-close", "days-before-issue-stale", "days-before-pr-close", "days-before-pr-stale", "days-before-stale", "debug-only", "delete-branch", "enable-statistics", "exempt-all-assignees", "exempt-all-issue-assignees", "exempt-all-issue-milestones", "exempt-all-milestones", "exempt-all-pr-assignees", "exempt-all-pr-milestones", "exempt-assignees", "exempt-draft-pr", "exempt-issue-assignees", "exempt-issue-labels", "exempt-issue-milestones", "exempt-milestones", "exempt-pr-assignees", "exempt-pr-labels", "exempt-pr-milestones", "ignore-issue-updates", "ignore-pr-updates", "ignore-updates", "include-only-assigned", "labels-to-add-when-unstale", "labels-to-remove-when-stale", "labels-to-remove-when-unstale", "only-issue-labels", "only-issue-types", "only-labels", "only-pr-labels", "operations-per-run", "remove-issue-stale-when-updated", "remove-pr-stale-when-updated", "remove-stale-when-updated", "repo-token", "sort-by", "stale-issue-label", "stale-issue-message", "stale-pr-label", "stale-pr-message", "start-date"

(action)